release-notes/Cube/Cube_Feature_Release_10.3/Cube_10.3.1.md

```diff
@@ -26,6 +26,14 @@ This setting allows you to specify the maximum amount of memory that SLAnalytics
 
 Default value: 2.00 GB
 
+#### DataMiner Cube - Visual Overview: Session variable YAxisResources now supports filters [ID_34857]
+
+<!-- MR 10.4.0 - FR 10.3.1 -->
+
+From now on, you can pass filters to the YAxisResources session variable in order to show the corresponding resource bands.
+
+The number of resources shown on a timeline is limited to 100 resources to avoid delay while loading the bands.
```

Comment:
There is no software limit of 100 resources.
Better would be to say that is recommended to use a filter which results in less than 100 resources.